app/assets/javascripts/oxalis/view/settings/dataset_settings_view.js

```diff
@@ -120,6 +131,11 @@ class DatasetSettings extends React.PureComponent<DatasetSettingsProps> {
             <Option value="1">medium</Option>
             <Option value="2">low</Option>
           </DropdownSetting>
+          <SwitchSetting
```

Comment:
In my opinion, it's not clear what is going to happen when disabling this setting. We should point out that the alternative to "Render Missing Data Black" is (something like) "Upsample lower resolution data for missing higher resolution data". Maybe use an info tooltip next to the setting :)